src/async_fuse/memfs/dir.rs

```diff
@@ -132,6 +132,12 @@ impl DirEntry {
         self.file_attr.read().kind
     }
 
+    /// Set name of the `DirEntry`.
+    #[allow(dead_code)]
```

Comment:
Did you forget to remove this 🤔 ?

Reply:
No.

This method is called by `rename_exchange_helper`, but it's only available in 7.23. In the ABI version under 7.23, the method `set_entry_name` is not called, that's why I added `#[allow(dead_code)]` to it. And this method is not ABI specified, so I can't add `#[cfg(feature = "abi-7-23")]` to it. That's why I reserved `#[allow(dead_code)]`  here.

Reply:
Well, I removed this method, and impl it in another way.